apollo-gradle-plugin-kotlin/src/main/kotlin/com/apollographql/apollo/gradle/kotlin/ApolloExtension.kt

```diff
@@ -0,0 +1,117 @@
+package com.apollographql.apollo.gradle.kotlin
+
+import com.apollographql.apollo.compiler.NullableValueType
+import org.gradle.api.Action
+
+open class ApolloExtension {
+
+  var nullableValueType = NullableValueType.ANNOTATED.value
+  var useSemanticNaming = true
+  var generateModelBuilder = false
+  var useJavaBeansSemanticNaming = false
+  var suppressRawTypesWarning = false
```

Comment:
As this plugin for Kotlin projects should we remove these codegen configurations:
```
generateModelBuilder
useJavaBeansSemanticNaming
suppressRawTypesWarning
generateVisitorForPolymorphicDatatypes
```

Reply:
I was thinking the plugin could be used for java projects as well since we want to continue supporting java ?